src/test/java/testsuite/integration/host/AuroraIntegrationContainerTest.java

```diff
@@ -82,19 +84,21 @@
   private static Network network;
 
   private static final ContainerHelper containerHelper = new ContainerHelper();
+  private static final AuroraTestUtility auroraUtil = new AuroraTestUtility();
 
   @BeforeAll
-  static void setUp() throws SQLException {
-    assertNotNull(DB_CONN_STR_SUFFIX, "DB_CONN_STR_SUFFIX should be set.");
-    assertTrue(DB_CONN_STR_SUFFIX.startsWith("."), "DB_CONN_STR_SUFFIX should start with period.");
-    assertNotNull(TEST_DB_CLUSTER_IDENTIFIER, "TEST_DB_CLUSTER_IDENTIFIER should be set.");
-    assertNotNull(TEST_USERNAME, "TEST_USERNAME should be set.");
-    assertNotNull(TEST_PASSWORD, "TEST_PASSWORD should be set.");
+  static void setUp() throws SQLException, InterruptedException {
+    if (StringUtils.isNullOrEmpty(TEST_DB_CLUSTER_IDENTIFIER) || StringUtils.isNullOrEmpty(TEST_USERNAME) || StringUtils.isNullOrEmpty(TEST_PASSWORD)) {
+      dbConnStrSuffix = auroraUtil.initCluster(); // Using default values
+    } else {
+      dbConnStrSuffix = auroraUtil.initCluster(TEST_USERNAME, TEST_PASSWORD, TEST_DB_CLUSTER_IDENTIFIER);
+    }
 
+    dbConnStrSuffix = dbConnStrSuffix.substring(dbConnStrSuffix.indexOf('.'));
     final String dbConnHostBase =
-        DB_CONN_STR_SUFFIX.startsWith(".")
-            ? DB_CONN_STR_SUFFIX.substring(1)
-            : DB_CONN_STR_SUFFIX;
+        dbConnStrSuffix.startsWith(".")
```

Comment:
I think we can make sure that auroraUtil returns a proper value (without a leading period) so we can avoid this logic,